ui/src/app/cron-workflows/components/cron-workflow-spec-editior.tsx

```diff
@@ -12,6 +13,7 @@ export const CronWorkflowSpecEditor = ({onChange, spec}: {spec: CronWorkflowSpec
                     <div className='columns small-3'>Schedule</div>
                     <div className='columns small-9'>
                         <TextInput value={spec.schedule} onChange={schedule => onChange({...spec, schedule})} />
+                        <ScheduleValidator schedule={spec.schedule} />
```

Comment:
I think this should be inline with the input, not in a new line. The spec string will never be long enough to occupy even a tenth of the length of the input box, so no reason we can't use the remaining length more effectively with the validator

Reply:
I'd like to leave this, it is a small panel and I don't think it should be optimised.